components/sidebar/sidebar.jsx

```diff
@@ -21,8 +21,8 @@ import * as ChannelUtils from 'utils/channel_utils.jsx';
 import {ActionTypes, Constants} from 'utils/constants.jsx';
 import * as Utils from 'utils/utils.jsx';
 import {t} from 'utils/i18n';
-import favicon from 'images/favicon/favicon-16x16.png';
-import redFavicon from 'images/favicon/redfavicon-16x16.png';
+import favicon from 'images/favicon/R_16x16.png';
+import redFavicon from 'images/favicon/R_16x16.png';
```

Comment:
This change seems to lose functionality. Did you let Amy know that the favicon was used to indicate if there are notifications, so that we probably should have 2 different ones?

Reply:
Right, I will ask Amy about this. What is the best Slack channel to address these types of things in? Or is a direct message better?